fints/utils.py

```diff
@@ -1,8 +1,106 @@
 import mt940
+import re
+from .models import Holding
+from datetime import datetime
 
 
 def mt940_to_array(data):
     data = data.replace("@@", "\r\n")
     data = data.replace("-0000", "+0000")
     transactions = mt940.models.Transactions()
     return transactions.parse(data)
+
+def print_segments(message):
+    segments = str(message).split("'")
+    for idx, seg in enumerate(segments):
+        print(u"{}: {}".format(idx, seg.encode('utf-8')))
+
+class MT535_Miniparser:
+
+    def parse(self, lines):
+        retval = []
+        re_identification = re.compile(r"^:35B:ISIN\s(.*)\|(.*)\|(.*)$")
+        re_marketprice = re.compile(r"^:90B::MRKT\/\/ACTU\/([A-Z]{3})(\d*),{1}(\d*)$")
+        re_pricedate = re.compile(r"^:98A::PRIC\/\/(\d*)$")
+        re_pieces = re.compile(r"^:93B::AGGR\/\/UNIT\/(\d*),(\d*)$")
+        re_totalvalue = re.compile(r"^:19A::HOLD\/\/([A-Z]{3})(\d*),{1}(\d*)$")
```

Comment:
You could move those to be direct properties of the class, so they don't need to be recompiled every time.
```
class MT535_Miniparser:
    re_identification = re.compile(r"^:35B:ISIN\s(.*)\|(.*)\|(.*)$")
    ...

    def …:
        self.re_identification.match(…)
```

But it probably doesn't matter, because it isn't expected to be called that often.